lib/views/movie_list_view.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
+
+import '../repositories/movie_repository.dart';
+import '../text_styles/custom_text_style.dart';
+import '../ui_constants.dart';
+import 'movie_details_view.dart';
+import '../entities/movie.dart';
+
+class MovieListView extends StatefulWidget {
+  const MovieListView({super.key});
+
+  @override
+  State<MovieListView> createState() => _MovieListViewState();
+}
+
+class _MovieListViewState extends State<MovieListView> {
+  final MovieRepository movieRepository = MovieRepository();
+  static const double imageHeight = 400;
+  static const double dividerHeight = 20;
+
+  @override
+  Widget build(BuildContext context) {
+    return FutureBuilder(
+      builder: (context, AsyncSnapshot snapshot) {
```

Comment:
I recommend to specify the type of the parameter in this case you have BuildContext and AsyncSnapshot<List<Movie>>, this allow you to avoid the casting on line 26

Reply:
done